c/cytoolz/build_info.json

```diff
@@ -15,8 +15,12 @@
     "dir": "ubi8.7",
     "build_script": "cytoolz_ubi_8.7.sh"
   },
-  "*": {
+  "0.12.0": {
     "dir": "ubi9.3",
     "build_script": "cytoolz_ubi_9.3.sh"
+  },
+  "*": {
```

Comment:
Update docker_build : "true" as you are adding dockerfile and also update the maintainer name on this file and "*" always need to take the default build-script which is cytoolz_ubi_9.3.sh and the version which you worked 0.12.0 which need to point to cytoolz_1.0.1_ubi_9.5.sh